src/frontend-pwa/src/components/utility/BannerTip/BannerTip.tsx

```diff
@@ -12,28 +12,21 @@ import {
   StyledToolTip,
 } from './bannerTip.style';
 import clearicon from '/iconography/ClearWhite.svg';
+import useAppService from '../../../services/app/useAppService';
 
-type BannerProps = {
-  text: string;
-  setText: (text: any) => void;
-  seconds: number
-}
+function BannerTip() {
+  const { state, setToolTipText } = useAppService();
 
-function BannerTip({
-  text,
-  setText,
-  seconds,
-}: BannerProps) {
-  const handleClear = () => { setText(''); };
+  const handleClear = () => { setToolTipText(''); };
   useEffect(() => {
-    if (text) {
-      setTimeout(handleClear, (seconds * 1000));
+    if (state.toolTipText) {
+      setTimeout(handleClear, (15000));
```

Comment:
A Horie level nitpick, but seconds should be a constant at the top 
`const seconds = 15` and then do a 
`* 1000` in here with the constant so it is more human readable
Doing so lets you manipulate the entire banner (math free) by flipping the constant at the top.

Reply:
@LocalNewsTV Done :)